File: workspace/data-ingestion/src/main/java/de/pfiva/data/ingestion/model/Feedback.java
package de.pfiva.data.ingestion.model;

import java.time.LocalDateTime;

public class Feedback {

	private int id;
	private String feedbackQuery;
	private String userResponse;
	private LocalDateTime timestamp;
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getFeedbackQuery() {
		return feedbackQuery;
	}
	public void setFeedbackQuery(String feedbackQuery) {
		this.feedbackQuery = feedbackQuery;
	}
	public String getUserResponse() {
		return userResponse;
	}
	public void setUserResponse(String userResponse) {
		this.userResponse = userResponse;
	}
	public LocalDateTime getTimestamp() {
		return timestamp;
	}
	public void setTimestamp(LocalDateTime timestamp) {
		this.timestamp = timestamp;
	}
	
	@Override
	public String toString() {
		return "Feedback [id=" + id + ", feedbackQuery=" + feedbackQuery + ", userResponse=" + userResponse
				+ ", timestamp=" + timestamp + "]";
	}
}

File: angular_workspace/pfiva-web/src/app/data-model/Feedback.ts
export class Feedback {

    private _id: number;
    private _feedbackQuery: string;
    private _userResponse: string;
    private _timestamp: string;

    public get id(): number {
		return this._id;
    }
    
	public set id(value: number) {
		this._id = value;
	}
    
	public get feedbackQuery(): string {
		return this._feedbackQuery;
	}

	public set feedbackQuery(value: string) {
		this._feedbackQuery = value;
	}

	public get userResponse(): string {
		return this._userResponse;
	}

	public set userResponse(value: string) {
		this._userResponse = value;
	}

	public get timestamp(): string {
		return this._timestamp;
	}

	public set timestamp(value: string) {
		this._timestamp = value;
	}
}
